### PR TITLE
Commands description strings

### PR DESCRIPTION
**Pull request by FnrDev**
**Please describe the changes this PR makes and why it should be merged:**

**In this PR:**
- Strings were added/removed (new/deleted string keys)

### CODE REVIEW

**Comment by FnrDev** [comment]
There are missing one string, for `general description` if anyone has a suggestion please let me know.

File: en/website.json [hunk 1]
```diff
@@ -709,6 +709,14 @@
     "premium_invite_required": "You have to invite the premium bot to get access to premium features",
     "notifications_send_channel": "Send To Channel",
     "notifications_content": "Insert Content Here",
+    "commands_title_description": "New to ProBot and wanna see all commands and their usages, also a little title describes it? You are in the right place!",
+    "commands_all_description": "All Commands",
+    "commands_general_description": "",
```
Comment (Dramex) [inline en/website.json:714]:
```suggestion
    "commands_general_description": "Useful commands every server must have!",
```

File: en/website.json [hunk 2]
```diff
@@ -709,6 +709,14 @@
...
+    "commands_social_description": "View your social details and share them with your friends using these commands!",
```
Comment (Dramex) [inline en/website.json:715]:
```suggestion
    "commands_leveling_description": "Boost up your server with text and voice leveling",
```

File: en/website.json [hunk 3]
```diff
@@ -709,6 +709,14 @@
...
+    "commands_info_description": "Want to see statistics / info about your server or users? Check these commands.",
+    "commands_moderation_description": "Power up moderation with these cool commands!",
+    "commands_music_description": "Enjoy high-quality music with more than 10+ commands to control your BEATS!",
+    "commands_premium_description": "Control your custom bot with these special commands!"
```
Comment (Dramex) [inline en/website.json:719]:
```suggestion
    "commands_premium_description": "Control your premium bot with these perks!"
```

**Review by Dramex — COMMENTED** [review]
owo

**Pull request merged**